VelodyneBlock[12] blocks
uint32 velodyne_timestamp
uint16 reserved
